packages/github/src/init.ts

```diff
@@ -53,7 +53,7 @@ async function getPrReleaseType(context: IContext, config: IPluginConfig): Promi
     const releaseLabels = Array.isArray(config.checkPrLabels) ? config.checkPrLabels : DEFAULT_RELEASE_LABELS;
     let approvedLabelEvents = await findApprovedLabelEvents(context, octokit, prNumber, releaseLabels);
 
-    if (approvedLabelEvents.length !== 1 && !context.dryRun) {
+    if (approvedLabelEvents.length !== 1 && !context.dryRun && context.branch.level !== "none") {
```

Comment:
This is the only instance that we seem to have hardcoded a value from the `SemverDiffLevels` array. 🤔 

Q: Should we replace `none` here for `SemverDiffLevels[0]` ?

Personal opinion: No. Even though we may never add something before/below `none` I rather not have to remember to change this check.

The other option could be to check for `context.branch.level > SemverDiffLevels.indexOf("none")`. But that means that anything below/before `none` follows the same rules.